docs/markdown/Reference-manual.md

```diff
@@ -1798,6 +1798,23 @@ an external dependency with the following methods:
 
  - `version()` is the version number as a string, for example `1.2.8`
 
+ - `partial_dependency(compile_args : false, link_args : false, links : false,
+   includes : false, source : false)` (*added 0.46.0) returns a new dependency
+   object with the same name, version, found status, type name, and methods as
+   the object that called it. This new object will only inherit other
+   attributes from it's parent as controlled by keyword arguments.
```

Comment:
"It's" means either "it is" or "it has". The possessive form of it is "its".

Reply:
gah, even as a native English speaker `it's` vs `its` makes no sense...

Reply:
I think it's harder for native speakers than for people who learnt English as a second language :)